app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -576,6 +581,25 @@ object TelemetryWrapper {
         TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.MENU, Value.RELOAD).queue();
     }
 
+    @JvmStatic
+    fun SSLError(fromPage: Boolean, error: SslError) {
+        val primaryErrorMessage:String
+        when (error.primaryError) {
```

Comment:
nit: You can use `when` directly in an assignment:

```
val x = when(y) {
  a -> ".."
  b -> ".."
}
```

Reply:
Very cool!